### ISSUE TITLE
Colorize SQL colors code incorrectly if there's any multi-line comment which is opened and closed in the same line

### ISSUE THREAD

**Issue by rdomene** [stevencohn/OneMore#634]
**Describe the bug**
Colorize SQL colors incorrectly when there are multi-line comments that are opened and closed in the same line. It colorizes as a comment all the lines between the first "/*" string and the last "*/" string that does not have another "/*" in the same line.

**To Reproduce**
1. Go to any page.
2. Paste the following SQL code:

```
SELECT
	GETDATE() AS [Today],
	/* This is a multi-line comment that starts and ends in the same line */
	(GETDATE() + 1) AS [Tomorrow],
	CAST(1 as bit) AS [Boolean],
	/*This is another multi-line comment that starts and ends in the same line */
	CAST(1 as int) AS [Number]
GO

/*
** SQL 2
*/

SELECT
	GETDATE() AS [Today],
	/* This is a multi-line comment that starts and ends in the same line */
	(GETDATE() + 1) AS [Tomorrow],
	CAST(1 as bit) AS [Boolean],
	/*
	This is a multi-line comment that starts and ends in different lines
	*/
	CAST(1 as int) AS [Number]
GO
```

3. Select all the code.
4. Click on More > Colorize > SQL
5. Color pattern is incorrect as some non-comment lines are colorized as comments.

**Expected behavior**

Right color pattern should be similar to this one:

![right-sql-colors](https://user-images.githubusercontent.com/23736473/192727362-652f4aee-2cca-4d4d-82af-58e528e87602.PNG)

Obtained color pattern is this one:

![wrong-sql-colors](https://user-images.githubusercontent.com/23736473/192727500-c5b41ab6-b49f-4634-8e8f-e2230eae4249.PNG)

**Environment (if applicable):**
 - Microsoft® OneNote® 2016 MSO (version 2208 compilation 16.0.15601.20148) - 64 bits 
 - OneMore Version 5.1.0

**Comment by stevencohn** [comment]
This should be fixed in 5.1.0 and in fact, I cannot reproduce this given the text that you provided.

**Comment by rdomene** [comment]
Interesting.

I uninstalled and reinstalled OneMore, just in case.

Then I copied and pasted the SQL code I pasted as a sample. Just Copy & Paste. I selected the text, then clicked Colorize > SQL, and the text was properly colored.

![colorize-sql-1](https://user-images.githubusercontent.com/23736473/192980252-f63dd2a3-cc3b-4573-ab6b-78d216432172.PNG)

But... after that, I copied the code and pasted it with the Paste Special > Paste Keep Text Only option (this is the method I use most of the time), and then I select the text, click Colorize > SQL, then the coloring pattern is incorrect.

![colorize-sql-3](https://user-images.githubusercontent.com/23736473/192981600-e3e19694-4906-4fab-8da4-d54661b2caad.PNG)

![colorize-sql-2](https://user-images.githubusercontent.com/23736473/192980803-db784aaa-fc69-4f99-ad4c-cce8abd81d20.PNG)
